README.md

```diff
@@ -81,7 +77,8 @@ Finally, we also loaded the pre-specified hostmap configuration from file using.
 # load host map configuration from file
 config = tfe.config.load('config.json')
 ```
-Take a look at [`/tools/gcp/link`](./tools/gcp/link) as an example to generate the config file for gcp. If you run it locally, you can use simply `tfe.LocalConfig`. You can find an example [here](./examples/federated-average/run.py#L20).
```

Comment:
note that I removed line number reference here as things like that tend to change without being properly fixed up in these other places